examples/dt/uni05epsilon/nodeset2/values.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: edpm-nodeset2-values-post-ceph
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  ceph:
+    conf: _replaced_
+    keyring: _replaced_
+
+  nodeset:
+    services:
+      - bootstrap
+      - download-cache
+      - configure-network
+      - validate-network
+      - install-os
+      - ceph-hci-pre
+      - configure-os
+      - ssh-known-hosts
```

Comment:
ditto

Reply:
Based on the previous comments its it ok to resolve this? @bogdando